### ISSUE TITLE
"Unbale to connect to the server" on Sign In

### ISSUE THREAD

**Issue by fssmoura** [Tustin/PlayStationDiscord#156]
Trying to Sign In but always getting the "Unable to connect to the server" error.
Code is 18.1d0e08c3.1616266188.10ca2182.
Tried older versions, running as administrator, nothing works.

**Comment by daneobyrd** [comment]
I am trying to sign in (3.2) and got this error:
`A connection to the server could not be established.
(18.25ca4d17.1616280620.96265ec)`

**Comment by RobinZockt** [comment]
i have the same issue

**Comment by austonpramodh** [comment]
I am facing the same issue.

**Comment by Meowmel** [comment]
Same error. I went with the previous version: https://github.com/Tustin/PlayStationDiscord/releases/tag/v3.1.2
It allowed the anti-bot captcha to pop up. I guess the latest version broke the captcha window.

**Comment by mrandre21** [comment]
Trying to sign in into PlayStation Discord and getting the same issue with the 

`A connection to the server could not be established. (18.25ca4d17.1616280620.96265ec)`

**Comment by rxxxxxxx** [comment]
When i go back to previous version 3.1.2 it does not show me as online, only says im offline

**Comment by Smexhy** [comment]
Exactly the same issue on the latest macOS version.

**Comment by rxxxxxxx** [comment]
Apparently re-installing discord and playstation discord works, which it does not for me, I still get the same message.

**Comment by shadow05139** [comment]
I also am experiencing  this also now could be broken the captcha window or PSN have also added 2FA to PSN accounts.

**Comment by Shiftah** [comment]
I'm getting the same issue with version 3.2.0, no captcha is displayed after password entry like in previous versions, instead I get 

`A connection to the server could not be established. (18.96b0f748.1618175164.127eb351)`

Unfortunately I need the PS5 support so rolling back to 3.1.2 isn't a solution as after logging in to PSN that just shows me as permanently offline.

**Comment by BreakingTV** [comment]
Same here

**Comment by BreakingTV** [comment]
#159 He can't Change a Thing :(

**Comment by BreakingTV** [comment]
#86 maybe this helps :/

**Comment by Shiftah** [comment]
> #86 maybe this helps :/

Yeah no I'm experiencing the issue after he pushed the downgraded version of Electron. Have also tried logging in in 3.1.2 then upgrading to 3.2.0, this causes the issue reported in #160 forcing login which brings us back to this issue here.

**Comment by BreakingTV** [comment]
Ah okay I see

**Comment by Shiftah** [comment]
Ok weirdest thing, I uninstalled and resintalled (for like the 5th time in 24 hours), checked the box in the white banner at the top to allow Playstation to store cookies, then did my login and it suddenly worked, no captcha needed. Discord can now see when my PS5 is online :)

**Comment by BreakingTV** [comment]
I don't get it. Is it luck if it works or not or how is it working. Yea but good that it works for you.

I try more and if I find a result I write it here

**Comment by BreakingTV** [comment]
Sooo I got it

I am on linux so I explain what I have done for linux:
I deleted the PlaystationDiscord folder in the .config folder and installed the version 3.1.0 and it worked fine for me!

PS: DONT UPGRADE THE VERSION an error comes up with the Token by me when I update it

**Comment by BreakingTV** [comment]
Yea it didnt work again now thanks :(

**Comment by theoneandonly33** [comment]
> Ok weirdest thing, I uninstalled and resintalled (for like the 5th time in 24 hours), checked the box in the white banner at the top to allow Sony / Playstation to store cookies, then did my login and it suddenly worked, no captcha needed. Discord can now see when my PS5 is online :)

How did you uninstall? Did you use an another programme to uninstall or just through windows settings?

**Comment by Shiftah** [comment]
> > Ok weirdest thing, I uninstalled and resintalled (for like the 5th time in 24 hours), checked the box in the white banner at the top to allow Sony / Playstation to store cookies, then did my login and it suddenly worked, no captcha needed. Discord can now see when my PS5 is online :)
> 
> How did you uninstall? Did you use an another programme to uninstall or just through windows settings?

Just via _add or remove programs_ in Windows 10 settings. I redownloaded the installer today too, buit I'm not sure that should make a difference as I can't see anything would have changed since last night. Allowing cookies seems to have been what made it work for me.

**Comment by theoneandonly33** [comment]
> > > Ok weirdest thing, I uninstalled and resintalled (for like the 5th time in 24 hours), checked the box in the white banner at the top to allow Sony / Playstation to store cookies, then did my login and it suddenly worked, no captcha needed. Discord can now see when my PS5 is online :)
> > 
> > 
> > How did you uninstall? Did you use an another programme to uninstall or just through windows settings?
> 
> Just via _add or remove programs_ in Windows 10 settings. I redownloaded the installer today too, buit I'm not sure that should make a difference as I can't see anything would have changed since last night. Allowing cookies seems to have been what made it work for me.

ok thanks

**Comment by theoneandonly33** [comment]
> > > Ok weirdest thing, I uninstalled and resintalled (for like the 5th time in 24 hours), checked the box in the white banner at the top to allow Sony / Playstation to store cookies, then did my login and it suddenly worked, no captcha needed. Discord can now see when my PS5 is online :)
> > 
> > 
> > How did you uninstall? Did you use an another programme to uninstall or just through windows settings?
> 
> Just via _add or remove programs_ in Windows 10 settings. I redownloaded the installer today too, buit I'm not sure that should make a difference as I can't see anything would have changed since last night. Allowing cookies seems to have been what made it work for me.

I uninstalled and reinstalled, and i selected the box to allow cookies, but it didn't work same error, and if I reinstall the cookies box is no longer there

**Comment by theoneandonly33** [comment]
Ive figured something out which may help people. First, uninstall using the uninstaller or through windows settings, then install and use a program called "Search Everything". Now search "Discord" and highlight and delete everything playstation discord and permanently delete in Recycle Bin. Next Install Playstation Discord and open. 

1. There should be a cookies box at the top of the window, click "click here".
2. Now click "More Information"
3. You will be brought to Sony's website in Playstation Discord, signing in may fix through there.

**Comment by MALVItPro** [comment]
> Ive figured something out which may help people. First, uninstall using the uninstaller or through windows settings, then install and use a program called "Search Everything". Now search "Discord" and highlight and delete everything playstation discord and permanently delete in Recycle Bin. Next Install Playstation Discord and open.
> 
> 1. There should be a cookies box at the top of the window, click "click here".
> 2. Now click "More Information"
> 3. You will be brought to Sony's website in Playstation Discord, signing in may fix through there.

Hmm, I can't seem to find this cookies box. Are you talking about the latest version?

**Comment by theoneandonly33** [comment]
> > Ive figured something out which may help people. First, uninstall using the uninstaller or through windows settings, then install and use a program called "Search Everything". Now search "Discord" and highlight and delete everything playstation discord and permanently delete in Recycle Bin. Next Install Playstation Discord and open.
> > 
> > 1. There should be a cookies box at the top of the window, click "click here".
> > 2. Now click "More Information"
> > 3. You will be brought to Sony's website in Playstation Discord, signing in may fix through there.
> 
> Hmm, I can't seem to find this cookies box. Are you talking about the latest version?

Yes, you have to completely remove the installer and everything else on from the latest version, and reinstall the latest version and it should be there

**Comment by MaitreAnonyme** [comment]
> > > Ive figured something out which may help people. First, uninstall using the uninstaller or through windows settings, then install and use a program called "Search Everything". Now search "Discord" and highlight and delete everything playstation discord and permanently delete in Recycle Bin. Next Install Playstation Discord and open.
> > > 
> > > 1. There should be a cookies box at the top of the window, click "click here".
> > > 2. Now click "More Information"
> > > 3. You will be brought to Sony's website in Playstation Discord, signing in may fix through there.
> > 
> > 
> > Hmm, I can't seem to find this cookies box. Are you talking about the latest version?
> 
> Yes, you have to completely remove the installer and everything else on from the latest version, and reinstall the latest version and it should be there

I have accepted cookies, I have installed / uninstalled the software X times, and I am still blocked

**Comment by RagnarLothbrok-Odin** [comment]
Same issue here, downgraded to 3.1.2 and could log in fine after I solved a captcha. On latest update I get no captcha and the error 'A connection to the server could not be established'

**Comment by MaitreAnonyme** [comment]
Problem is, I'm no longer on PS4, now I'm on PS5
Sorry for Google Translate

**Comment by Tenshigami** [comment]
I likewise am having this issue after I accidentally opened PlayStationDiscord while my internet was down yesterday morning, causing it to require me to log in again. I had previously never had to log into 3.2.0 as it had remembered my login from a previous version. Now I get the error when trying to log in with 3.2.0. I can log in with 3.1.2, but upgrading seemingly causes an error with refreshing the token, forcing me to log in again.

I'll stick with 3.1.2 for now but as I recently managed to get my hands on a PS5, and PS5 support was added in 3.2.0, this is not ideal.

**Comment by TheRadziu** [comment]
same issue here, tried compiling the development branch and this error is still present there.

**Comment by V33m** [comment]
@Tustin have you looked into the issue concerning logging in when 2FA is enabled? Do you need any help or input to figure it out? I'm facing the same issue on both Windows and MacOS.

**Comment by V33m** [comment]
Unfortunately, the news Tustin is sharing is not very promising ([Tustin's Twitter account](https://twitter.com/Tusticles/status/1387069699649851397?s=20))

"I've had a lot of people asking me about hot to fix the connection issue when trying to login with PlayStationDiscord. 

Unfortunately this is an issue with Sony's login form and not necessarily an issue with the program. The error occurs when Sony thinks you are a bot.

I've tried to fix this issue numerous times with no luck. It's hard to figure out what exactly they're detecting. It also sometimes will randomly work while for others they are never able to login Person shrugging"

**Comment by V33m** [comment]
Unfortunately, the news Tustin is sharing is not very promising ([Tustin's Twitter account](https://twitter.com/Tusticles/status/1387069699649851397?s=20))

> I've had a lot of people asking me about hot to fix the connection issue when trying to login with PlayStationDiscord. 
> 
> Unfortunately this is an issue with Sony's login form and not necessarily an issue with the program. The error occurs when Sony thinks you are a bot.
> 
> I've tried to fix this issue numerous times with no luck. It's hard to figure out what exactly they're detecting. It also sometimes will randomly work while for others they are never able to login

**Comment by Tenshigami** [comment]
^ I just went ahead and tried again with the new version and it worked, without even giving me a captcha. This was after not even using any version of PlayStationDiscord at all for the last 10+ days. I wonder if there's something Sony flags on your account when the bot detection catches something that makes it not work, which then gets removed after a number of hours/days.
